docs/src/model_setup/boundary_conditions.md

```diff
@@ -1,62 +1,173 @@
 # [Boundary conditions](@id model_step_bcs)
 
-A boundary condition is applied to each field, dimension, and endpoint. There are left and right boundary conditions
-for each of the x, y, and z dimensions so each field has 6 boundary conditions. Each of these boundary conditions may
-be specified individually. Each boundary condition can be specified via a constant value, an array, or a function.
-
-The left and right boundary conditions associated with the x-dimension are called west and east, respectively. For the
-y-dimension, left and right are called south and north. For the z-dimension, left and right are called bottom and top.
+Boundary conditions are intimately related to the grid topology, and only
+need to be considered in directions with `Bounded` topology.
```

Comment:
That's not true if there are immersed solids and IBM is used, no?

Reply:
I was thinking that immersed boundaries could only be used with `Bounded` but you're right that we do have `GridFittedBoundary` for 3D stuff which can be used with `Periodic`. I'll update.